src/main/java/org/opensearch/neuralsearch/common/VectorUtil.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.neuralsearch.common;
+
+import java.util.List;
+
+/**
+ * Utility class for working with vectors
+ */
+public class VectorUtil {
```

Comment:
nit pick, you can add @NoArgsConstructor(accessLevel=PRIVATE) on this class to make sure that we are not creating instance of this class anywhere

Reply:
Good suggestion - will update